content/blog/2017/2017-02-06-declarative-pipeline-ga.adoc

```diff
@@ -0,0 +1,89 @@
+1.0 to
+link:https://plugins.jenkins.io/workflow-aggregator[Jenkins Pipeline.]
+We think this new syntax will enable everyone involved in DevOps, regardless of expertise,
+to participate in the continuous delivery process. Whether creating, editing or reviewing
+a pipeline, having a straightforward structure helps to understand and predict the
+flow of the pipeline and provides a common foundation across all pipelines.
+
+= Pipeline as Code
+Pipeline as Code was one of the pillars of the Jenkins 2.0 release and an essential part of implementing continuous delivery. Defining all of the stages of an application's CD pipeline within a "Jenkinsfile" and treating it as part of the application code automatically provides all of the benefits inherent in SCM:
+
+* Retain history of all changes to Pipeline
+* Rollback to a previous pipeline version
+* Review new changes to the pipeline in code review
+* Test new pipeline steps in branches
+* Audit changes to the pipeline
+* Run the same pipeline on a different Jenkins server
+
+= Getting Started with Declarative Pipeline
+We recommend people begin using it for all their Pipeline definitions in Jenkins.
+The plugin has been available for use and testing starting with the 0.1 release that was debuted at
+link:https://www.cloudbees.com/introducing-new-way-define-jenkins-pipelines[Jenkins World]
+in September, it has already been installed over 5,000 times.
+
+If you haven't tried Pipeline or have considered Pipeline in the past we believe
+this new syntax is much more approachable with an easy adoption curve to quickly
+realize all of the benefits of Pipeline as Code. In addition, the predefined structure
+of Declarative makes it possible to create and edit pipelines with a graphical
+user interface (GUI). The Blue Ocean team is actively working on a Pipeline Editor
+that will be included in an upcoming release.
+
+If you have already begun using Pipelines in Jenkins we believe that this new alternative
+syntax can help expand that usage. The original syntax for defining pipelines in Jenkins
+is a Groovy DSL that allows most of the features of full imperative programming.
+This syntax is still fully supported and is now referred to as "Scripted Pipeline Syntax"
+to distinguish it from "Declarative Pipeline Syntax." Both use the same underlying execution
+engine in Jenkins and both will generate the same results in Pipeline Stage View or Blue Ocean
+visualization. All existing pipeline steps, global variables and shared libraries can be used
+in either. You can now create more cookie-cutter pipelines and extend the power of Pipeline to all users regardless of Groovy expertise.
+
+= Declarative Pipeline Features
+
+* link:https://en.wikipedia.org/wiki/Lint_%28software%29[Syntax Checking]
+** Immediate runtime syntax checking with explicit error messages
+** API endpoint for linting Jenkinsfiles
+** CLI command to lint Jenkinsfiles
```

Comment:
I didn't know about these features, but I did manage to find some documentation on the plugin's GitHub wiki.

Perhaps a link [to that page](https://github.com/jenkinsci/pipeline-model-definition-plugin/wiki/Validating-(or-linting)-a-Declarative-Jenkinsfile-from-the-command-line) would be useful here.